README.md

```diff
@@ -65,6 +65,19 @@ The Wallet SDK iOS xcframework packages are distributed through Swift Package Ma
 [wallet-sdk tags](https://github.com/trustbloc/wallet-sdk/tags) with the suffix `-swift-pm` (e.g., `1.0.0-swift-pm`) for the
 latest releases.
 
+### Web
+The Wallet SDK JS package is distributed through npm. Please refer to 
+[wallet-sdk-js package](https://github.com/trustbloc-cicd/snapshot/pkgs/npm/wallet-sdk-js) for the latest releases.
+
+#### Web Config
+You need to provide the path to the location where you'd store the wasm build file (`wallet-sdk.wasm`) through
```

Comment:
Can we add a sentence describing how the consumer can get the wam file from the npm package?